### PR TITLE
Add imgviz

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@wkentaro would you like to maintain this package?

File: recipes/imgviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - imgviz
+
+about:
+  home: https://github.com/wkentaro/imgviz
+  # Actually, the license is unkown, I simply took one from a similar project
+  # from the same author
+  # https://github.com/wkentaro/labelme
+  # Upstream asked for clarification:
+  # https://github.com/wkentaro/imgviz/issues/11
+  license: GPL-3.0
```
Comment (npavlovikj) [inline recipes/imgviz/meta.yaml]: The license type listed on PyPI is MIT, https://pypi.org/project/imgviz/, and the setup file has listed it as MIT as well, https://github.com/wkentaro/imgviz/blob/master/setup.py.

Comment (hmaarrfk) [reply recipes/imgviz/meta.yaml]: Thanks! I updated it as much as I could. Upstream should definitely clarify.

Comment (npavlovikj) [reply recipes/imgviz/meta.yaml]: Thanks Mark! Let's give the developer some time to reply then.

**Comment by wkentaro** [comment]
@hmaarrfk Sure! Thanks for working on this.

**Pull request merged**